src/CodeGen_OpenGL_Dev.cpp

```diff
@@ -250,6 +250,14 @@ void CodeGen_GLSL::visit(const FloatImm *op) {
     id = oss.str();
 }
 
+void CodeGen_GLSL::visit(const IntImm *op) {
+    print_assignment(op->type, print_type(op->type) + "(" + std::to_string(op->value) + ")");
```

Comment:
I'm open to doing that-- this syntax is correct in C++ I believe but not C, but since CodeGen_C generates C++ this isn't a problem.  Unless anyone objects, I'll update the PR to change CodeGen_C instead.